### PR TITLE
Support treegraph and treemap modules

### PR DESCRIPTION
**Pull request by charlesfries**
https://www.highcharts.com/docs/chart-and-series-types/treemap
https://www.highcharts.com/docs/chart-and-series-types/treegraph-chart

### CODE REVIEW

**Comment by charlesfries** [comment]
@RobbieTheWagner is anything I can do to help get this PR and also #481 merged so all chart types are supported?

**Review by RobbieTheWagner — APPROVED** [review]
@charlesfries this looks good to me! I assume you have tested and the chart works? We may want to add this chart type to the demo page as well.

**Comment by charlesfries** [comment]
Will add a demo, give me a few minutes

**Comment by charlesfries** [comment]
Added the demo.

File: ember-highcharts/src/components/high-charts.ts [hunk 1]
```diff
@@ -243,6 +243,19 @@ export default class HighCharts extends Component<HighChartsSignature> {
       SolidGauge.default(Highcharts);
     }
 
+    if (
+      this.args.chartOptions?.chart?.type === 'treegraph' ||
```
Comment (RobbieTheWagner) [inline ember-highcharts/src/components/high-charts.ts:247]: So treegraph and treemap both require `Treemap`?

**Review by RobbieTheWagner — APPROVED** [review]

File: ember-highcharts/src/components/high-charts.ts (hunk 1 above)
Comment (charlesfries) [reply ember-highcharts/src/components/high-charts.ts:247]: Yes.


Treemap
https://www.highcharts.com/docs/chart-and-series-types/treemap
> The treemap chart requires the filemodules/treemap.js.

Treegraph
https://www.highcharts.com/docs/chart-and-series-types/treegraph-chart
> In order to use this series type, you need to load the modules/treemap.js and modules/treegraph.js modules.

**Pull request merged**